packages/google-pay-integration/.eslintrc.json

```diff
@@ -6,7 +6,9 @@
             "files": ["*.ts", "*.tsx", "*.js", "*.jsx"],
             "rules": {
                 "@typescript-eslint/naming-convention": "off",
-                "@typescript-eslint/no-unsafe-argument": "off"
+                "@typescript-eslint/no-unsafe-argument": "off",
+                "@typescript-eslint/no-unnecessary-condition": "off",
```

Comment:
We use the same rule in other packages to avoid `Unsafe assignment of an 'any' value` error when executing eslint. We need this rule in this package too

Reply:
I mean `@typescript-eslint/no-unnecessary-condition`

Reply:
I removed this rule (because it is unnecessary for now) and added another, but for a similar reason. Otherwise we use these rules to prevent lint errors